include/d/d_drawlist.h

```diff
@@ -171,6 +171,11 @@ struct dDlst_alphaModel_c {
     void create(int);
     void set(unsigned char, float(*)[4], unsigned char);
     void draw(float(*)[4]);
+public:
+    short pad;
+    short pad2;
+    short pad3;
+    short mCount;
```

Comment:
offset comments to left of members are preferred, and if members are temporarily unknown maybe just call them `field_0x` + offset num.
so something like
```cpp
/* 0x0 */ s16 field_0x0;
/* 0x2 */ s16 field_0x2;
/* 0x4 */ s16 field_0x4;
/* 0x6 */ s16 mCount;
```